package.json

```diff
@@ -1,7 +1,8 @@
 {
   "name": "kiloview-encoder",
-  "version": "1.0.0",
+  "version": "1.0.1",
   "api_version": "1.0.0",
+	"type": "module",
   "keywords": [
```

Comment:
Please remove unused properties, which are now in manifest.

Reply:
Removed all duplicate entries in package.json, except type is used by the project, so it has to be there